### ISSUE TITLE
Feedback - Technical Steps

### ISSUE THREAD

**Issue by steffilazerte** [ropensci-org/blog-guidance#93]
**Zero Step**  
In the [Technical Guidelines](https://blogguide.ropensci.org/technical.html) perhaps there should be a Zero step to first get the go-ahead:

0. First get the go-ahead and a tentative publication date from our community manager Stefanie Butland.

Just in case people stumble upon this book without first being directed there. Alternatively, add a box to the Preamble with something like (“Interested in Contributing a Blog Post or Tech Note? Contact Stefanie Butland!)

**`roblog`**
I would also consider listing the `roblog` package here (with a link) prominently

**Comment by stefaniebutland** [comment]
Question for Steffi: Where specifically should I list roblog?

The process at [Briefly, the process is:](https://blogguide.ropensci.org/technical.html) is meant to be generic with no links to specifics.

I could change the order of 2.2.1, .2, .3 to make roblog first option.

**Comment by steffilazerte** [comment]
I'd be tempted to add a small section on roblog right after the "Briefly the process is" (before 2.1). Something along the lines of:

Using the roblog package
- the roblog package is an option that can help with these steps
- if you want to use it, here's how to install

Then a) when the package is mentioned in the steps it's not a surprise, and b) it's clear where the installation instructions are in the TOC (I had a hard time finding them when I went back to install it, because I had to remember which step had the installation instructions). 

I think this would facilitate users jumping around the TOC and would make it more of a look-up guide rather than a book?
